### DISCUSSION TITLE
Match for Android Keystore

### DISCUSSION THREAD

**Discussion by bensig** [fastlane/fastlane#17058] [Ideas]
I saw requests dating back to 2016 asking for the capability to sign an android app using match. 

This would be a great addition to match since Android makes up such a large market share. 

Have you guys considered adding the capability to store the keystore and json in the certs repo again?

**Comment by KBSchmidt** [comment]
https://github.com/christopherney/fastlane-plugin-match_keystore

It looks like an unofficial plugin is being worked on. A feature like this would be great for whitelabled apps with googla app siging shenanigans
